shared/app/nav.native.js

```diff
@@ -198,6 +198,8 @@ class MainNavStack extends Component<any, any> {
     const stacks = this._stackCache
       .set(props.routeSelected, props.routeStack)
       .toArray()
+      // The inner views are replaced with placeholders so lets not even both to render the siblings
```

Comment:
Actually, `_stackCache` is a map, and we call `set` with `props.routeSelected`, so filtering on `key === props.routeSelected` will only ever return `props.routeStack`. i.e. this can be

```
const stack = <CardStackShim key={props.routeSelected} hidden={false} ... stack={props.routeStack} />
```
right?

Reply:
yeah, i didn't look at this that closely. i just removed _stackCache entirely as it serves no purpose now